src/components/Settings/ProfilePopup.js

```diff
@@ -0,0 +1,214 @@
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Avatar,
+  Button,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  Slide,
+  Snackbar,
+  TextField,
+} from "@material-ui/core";
+import { getCurrentUserData } from "../../firebase/getCurrentUserData";
+import { updateCurrentUserData } from "../../firebase/updateCurrentUserData";
+import MuiAlert from "@material-ui/lab/Alert";
+import { storage } from "../../index";
+import Skeleton from "@material-ui/lab/Skeleton";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    backgroundColor: "white",
+    maxWidth: 500,
+    height: "100%",
+    padding: theme.spacing(2),
+  },
+  buttons: {
+    display: "flex",
+    justifyContent: "space-between",
+    padding: 0,
+    paddingTop: 20,
+  },
+  button: {
+    paddingLeft: 0,
+    paddingRight: 0,
+  },
+  profileImage: {
+    width: theme.spacing(10),
+    height: theme.spacing(10),
```

Comment:
Auch hier könnte man es etwas größer machen. Wir haben ja den Platz
```suggestion
    width: theme.spacing(15),
    height: theme.spacing(15),
```